templates/zerver/integrations/nagios.md

```diff
@@ -1,64 +1,60 @@
-`/etc/nagios3/conf.d/contacts.cfg`, doing something like:
+1.  Finally, add `zulip` to the `members` list for one or more of the
+    contact groups in the `CONTACT GROUPS` section of
+    `/etc/nagios3/conf.d/contacts.cfg`, doing something like:
 
-```
-define contactgroup {
-    contactgroup_name       admins
-    alias                   Nagios Administrators
-    members                 monitoring, zulip
-}
-```
+        define contactgroup {
+            contactgroup_name       admins
+            alias                   Nagios Administrators
+            members                 monitoring, zulip
+        }
 
-Once you’ve done that, reload your Nagios configuration using
-`/etc/init.d/nagios3 reload`.
+1.  Once you’ve done that, reload your Nagios configuration using
+    `/etc/init.d/nagios3 reload`.
 
-When your Nagios system makes an alert, you’ll see a message like the
-following, to the stream `nagios` (to change this, edit the arguments
-to `nagios-notify-zulip` in `/etc/nagios3/conf.d/zulip_nagios.cfg`)
-with a topic indicating the service with an issue.
+1.  When your Nagios system makes an alert, you’ll see a message like the
+    following, to the stream `nagios` (to change this, edit the arguments
+    to `nagios-notify-zulip` in `/etc/nagios3/conf.d/zulip_nagios.cfg`)
+    with a topic indicating the service with an issue.
 
 {!congrats.md!}
 
 ![Nagios bot message](/static/images/integrations/nagios/001.png)
 
-**Testing**
+### Testing
 
-If you have [external commands enabled in Nagios][1],
-you can generate a test notice from your Nagios instance by
-using the `Send custom service notification` command in the
-`Service Commands` section of any individual service’s page
-on your Nagios instance.
+1.  If you have [external commands enabled in Nagios][1],
+    you can generate a test notice from your Nagios instance by
+    using the `Send custom service notification` command in the
+    `Service Commands` section of any individual service’s page
+    on your Nagios instance.
 
-[1]: https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/extcommands.html
+    [1]: https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/extcommands.html
 
-**Troubleshooting**
+### Troubleshooting
 
-You can confirm whether you’ve correctly configured Nagios to run the
-Zulip plugin by looking for `SERVICE NOTIFICATION` lines mentioning
-zulip in `/var/log/nagios3/nagios.log`. You can confirm whether you’ve
-configured the Zulip plugin code correctly by running
-`/usr/local/share/zulip/integrations/nagios/nagios-notify-zulip`
-directly.
+1.  You can confirm whether you’ve correctly configured Nagios to run the
+    Zulip plugin by looking for `SERVICE NOTIFICATION` lines mentioning
+    zulip in `/var/log/nagios3/nagios.log`. You can confirm whether you’ve
+    configured the Zulip plugin code correctly by running
+    `/usr/local/share/zulip/integrations/nagios/nagios-notify-zulip`
+    directly.
```

Comment:
I don't think we want single-item numbered lists for these follow-on sections; they should just be normal paragraphs.

![image](https://user-images.githubusercontent.com/2746074/111387953-ca626200-866b-11eb-8f69-a5f3398d4f42.png)